File: CS/MVC_WebDashboard/App_Start/DashboardConfig.cs
using System.Web.Routing;
using DevExpress.DashboardWeb;
using DevExpress.DashboardWeb.Mvc;
using Storages;
using System.Web;

public class DashboardConfig {
    public static void RegisterService(RouteCollection routes) {
        routes.MapDashboardRoute();
        DashboardConfigurator.Default.SetDashboardStorage(new CustomDashboardFileStorage(HttpContext.Current.Server.MapPath("~/App_Data/Dashboards")));
    }
}

File: VB/MVC_WebDashboard/App_Start/DashboardConfig.vb
Imports Microsoft.VisualBasic
Imports System.Web.Routing
Imports DevExpress.DashboardWeb
Imports DevExpress.DashboardWeb.Mvc
Imports Storages
Imports System.Web

Public Class DashboardConfig
	Public Shared Sub RegisterService(ByVal routes As RouteCollection)
		routes.MapDashboardRoute()
		DashboardConfigurator.Default.SetDashboardStorage(New CustomDashboardFileStorage(HttpContext.Current.Server.MapPath("~/App_Data/Dashboards")))
	End Sub
End Class